inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.h

```diff
@@ -87,6 +87,14 @@ class MKLDNNMVNNode : public MKLDNNNode {
 
     static bool checkAxesSuitability(const std::shared_ptr<const ngraph::Node>&);
 
+    inline bool acrossChannels() {
```

Comment:
1) As soon as it is classical getter I would prefer `getAcrossChannels` naming
2) It should be const

same for `normalizeVariance`

Reply:
Fixed